man/org.freedesktop.resolve1.xml

```diff
@@ -507,20 +510,23 @@ node /org/freedesktop/resolve1 {
       <citerefentry project="man-pages"><refentrytitle>gethostname</refentrytitle><manvolnum>3</manvolnum></citerefentry>,
       but may differ if a conflict is detected on the network.</para>
 
-      <para><varname>DNS</varname> and <varname>DNSEx</varname> contain arrays of all DNS servers currently
-      used by <filename>systemd-resolved</filename>. <varname>DNS</varname> contains information similar to
-      the DNS server data in <filename>/run/systemd/resolve/resolv.conf</filename>. Each structure in the
-      array consists of a numeric network interface index, an address family, and a byte array containing the
-      DNS server address (either 4 bytes in length for IPv4 or 16 bytes in lengths for IPv6).
-      <varname>DNSEx</varname> is similar, but additionally contains the IP port and server name (used for
-      Server Name Indication, SNI). Both arrays contain DNS servers configured system-wide, including those
-      possibly read from a foreign <filename>/etc/resolv.conf</filename> or the <varname>DNS=</varname>
-      setting in <filename>/etc/systemd/resolved.conf</filename>, as well as per-interface DNS server
-      information either retrieved from
+      <para><varname>DNS</varname>, <varname>DNSEx</varname>, and <varname>ManagerDNS</varname> contain
+      arrays of all DNS servers currently used by <filename>systemd-resolved</filename>.
+      <varname>DNS</varname> contains information similar to the DNS server data in
+      <filename>/run/systemd/resolve/resolv.conf</filename>. Each structure in the array consists of a
+      numeric network interface index, an address family, and a byte array containing the DNS server
+      address (either 4 bytes in length for IPv4 or 16 bytes in lengths for IPv6).
+      <varname>DNSEx</varname> and <varname>ManagerDNS</varname> are similar, but additionally contains
+      the IP port and server name (used for Server Name Indication, SNI). These three properties
+      contain DNS servers configured system-wide, including those possibly read from a foreign
+      <filename>/etc/resolv.conf</filename> or the <varname>DNS=</varname> setting in
+      <filename>/etc/systemd/resolved.conf</filename>. <varname>DNS</varname> and
+      <varname>DNSEx</varname> also contain as well as per-interface DNS server information either
+      retrieved from
       <citerefentry><refentrytitle>systemd-networkd</refentrytitle><manvolnum>8</manvolnum></citerefentry>,
       or configured by external software via <function>SetLinkDNS()</function> or
       <function>SetLinkDNSEx()</function> (see above). The network interface index will be 0 for the
-      system-wide configured services and non-zero for the per-link servers.</para>
+      system-wide configured servers and non-zero for the per-link servers.</para>
```

Comment:
"will be 0 for servers configured system-wide, and non-zero for per-link servers" would read better.